app/helpers/frequencies_helper.rb

```diff
@@ -3,13 +3,15 @@ def valid_frequencies
     t("frequencies").map { |frequency, _config| frequency.to_s }
   end
 
-  def frequencies(options)
+  def frequencies(options = false)
```

Comment:
Could you help me understand this change? It's a bit strange to default an argument to a value of a different type (`options` would be a hash). Perhaps we can pass in `checked_frequency => nil` instead?

Reply:
That sounds like a more consistent option, I'll do that 👍

Reply:
We should be able to simplify this even further: using the original helper, but defaulting `options` to an empty hash, and passing nothing in the `new_frequency` view. 

The modified helper code works, but it's hard to understand because it no longer sets `checked` to `true` or `false` - now it's either a string or `nil`.

Reply:
D'oh. I totally misread how this worked the first time. Will fix now.

Reply:
@benthorner Changed - I think that should be ok now?